src/data/general.json

```diff
@@ -6,6 +6,15 @@
         "initialValue": true,
         "initializeOnMount": true,
         "description": "Be notified about known maintenance windows, new features and marketing annoucements for cloud.redhat.com"
+      }, {
+        "component": "text-field",
+        "label": "First name",
+        "name": "first-name"
+      },
+      {
+        "component": "text-field",
+        "label": "First name",
```

Comment:
Not sure what it does in the UI, but you have 2x First Name here. Maybe it should be Last name.

Reply:
This is just an example/testing file so we don't have to run API server all the time. But yeah, good point so others are not confused when using this repo.